index.md

```diff
@@ -13,7 +13,7 @@ technologies.
 
 We provide [workshops][], [documentation][scaling book], [weekly
 newsletters][], [original research][dashboard], [case studies and
-announcements][blog], and help facilitate improved relations between businesses
+announcements][blog], [analysis of Bitcoin software and services][compatibility], and help facilitate improved relations between businesses
```

Comment:
@jnewbery you may also want to remove this for the partial merge.  You can confirm that no links point to these pages by deleting the pages from the `_site/` directory and then manually running the htmlproofer command from the Makefile.  If there are any links pointing to the pages, it should print an error.